hive_setup.sh

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
+set -xe
+
+source logging.sh
+source common.sh
+source utils.sh
+source rhcos.sh
+source ocp_install_env.sh
+
+function generate_hive_manifest() {
+    cat > "ocp/${CLUSTER_NAME}/manifests.yaml" <<EOF
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ${CLUSTER_NAME}-pull-secret
+type: kubernetes.io/dockerconfigjson
+stringData:
+  .dockerconfigjson: |-
+    $(echo $PULL_SECRET | jq -c .)
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ${CLUSTER_NAME}-ssh-private-key
+stringData:
+  ssh-privatekey: |-
+$(cat ~/.ssh/id_rsa | sed 's/^/    /g')
+
+---
+apiVersion: hive.openshift.io/v1
+kind: ClusterDeployment
+metadata:
+  name: ${CLUSTER_NAME}
+  annotations:
+    # do not retry if first deploy fails
+    hive.openshift.io/try-install-once: "true"
+spec:
+  baseDomain: ${BASE_DOMAIN}
+  clusterName: ${CLUSTER_NAME}
+  controlPlaneConfig:
+    servingCertificates: {}
+  platform:
+    bareMetal:
+      libvirtSSHPrivateKeySecretRef:
+        name: ${CLUSTER_NAME}-ssh-private-key
+  provisioning:
+    installConfigSecretRef:
+      name: ${CLUSTER_NAME}-install-config
+    sshPrivateKeySecretRef:
+      name: ${CLUSTER_NAME}-ssh-private-key
+#    manifestsConfigMapRef:
+#      name: ${CLUSTER_NAME}-install-manifests
```

Comment:
I assume you're targeting 4.4 thus no install manifests needed.

Reply:
Yes, that's right. I haven't actually shown that this all works, yet, but I don't expect to need any extra manifests.